src/linux/processor.rs

```diff
@@ -179,12 +179,11 @@ impl Processor {
         guest_nice: u64,
     ) {
         fn min(a: u64, b: u64) -> f32 {
-            (if a == b {
-                1
-            } else if a > b {
-                a - b
-            } else {
-                b - a
+            use std::cmp::Ordering;
+            (match a.cmp(&b) {
```

Comment:
Not sure to understand how this is any better but whatever. :laughing:

Reply:
If you prefer to keep it i can disable the lint on that function instead.

Reply:
I have to admit that I don't really care. At least it's one less clippy lint to worry about.